samples/python/tutorial_code/imgProc/BasicGeometricDrawing/basic_geometric_drawing.py

```diff
@@ -6,10 +6,10 @@
 def my_ellipse(img, angle):
     thickness = 2
     line_type = 8
-
+    
     cv.ellipse(img,
-                (W / 2, W / 2),
-                (W / 4, W / 16),
+                (int(W / 2), int(W / 2)),
+                (int(W / 4), int(W / 16)),
```

Comment:
`W // 2` is better (integer division).

Reply:
Sounds good.  Changed based on feedback.